redesign/index.html

```diff
@@ -2,9 +2,207 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="author" content="@dwyl">
+    <meta name="keywords" content="do, what, you, love, dwyl, mvp, time, tracking, help, developers, code, frontend, backend, tooling, tools">
+    <meta name="robots" content="index, follow, noarchive">
+    <meta name="base" content="https://www.dwyl.com">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/dwyl.css">
     <title>dwyl</title>
   </head>
   <body>
-    <h1>Hello world!</h1>
+    <nav class="w-100 vh-100 h3-ns dwyl-bg-dark-grey tc white">
```

Comment:
Only partial support in IE for `vh` and I know this caused @RobStallion some issues on FREED - please test this in IE10 + 11  (@danwhy has a good way of doing this with saucelabs manual testing and https://ngrok.com/ )